plugin/trino-hive/src/main/java/io/trino/plugin/hive/acid/AcidOperation.java

```diff
@@ -23,11 +23,25 @@
 public enum AcidOperation
 {
     // UPDATE and MERGE will be added when they are implemented
-    NONE,
-    CREATE_TABLE,
-    DELETE,
-    INSERT,
-    UPDATE;
+    NONE(false),
+    CREATE_TABLE(false),
+    DELETE(true),
+    INSERT(true),
+    UPDATE(true),
+    ANALYZE(true),
+    /**/;
+
+    final boolean acidTransaction;
```

Comment:
nit: adding `isAcidTransaction` could be a preparatory commit (no changes requested)